app/src/main/java/com/eveningoutpost/dexdrip/profileeditor/ImportAapsProfile.java

```diff
@@ -71,7 +71,7 @@ public static void importAndSaveFromMap(final HashMap<String, Object> tmap) {
             if (Pref.getBooleanDefaultFalse("profile_import_sound")) {
                 BackgroundQueue.post(() -> JoH.playSoundUri(JoH.getResourceURI(R.raw.labbed_musical_chime)));
             }
-            JoH.static_toast_long("xDrip imported AAPS profile");
+            UserError.Log.e(TAG, "xDrip imported AAPS profile");
```

Comment:
This logs it as an error but I believe it is just an info message. Maybe `uel` or `ueh` is more appropriate.